packages/emotion/src/styleUtils/makeThemeVars.ts

```diff
@@ -54,7 +54,7 @@ function makeThemeVars<P extends string, V extends Partial<ThemeVarGroup>>(
 ) {
   const themeVars: Record<string, unknown> = {}
   ;(Object.keys(vars) as Array<keyof V>).forEach((variable) => {
-    themeVars[camelize(`${prefix}-${variable}`)] = vars[variable]
+    themeVars[camelize(`${prefix}-${String(variable)}`)] = vars[variable]
```

Comment:
TS 4.7 has a bit stricter checks in this case, its needed when `variable` is `Symbol` type